builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -255,6 +259,47 @@ node {
 
 }
 
+/**
+ * Check bucket existance and content in it. if yes then empty it for SLS delete
+ * @param  stage
+ * @return
+ */
+def checkBucket(stage) {
+	withCredentials([
+		[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: configLoader.AWS_CREDENTIAL_ID, secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']
+	]) {
+		try {
+			sh "aws configure set profile.cloud-api.region ${configLoader.AWS.REGION}"
+			sh "aws configure set profile.cloud-api.aws_access_key_id $AWS_ACCESS_KEY_ID"
+			sh "aws configure set profile.cloud-api.aws_secret_access_key $AWS_SECRET_ACCESS_KEY"					
+			
+			// Empty bucket
+			def targetBucket = service_config['event_source_s3'];
+			if(stage != "prod"){
+				targetBucket = service_config['event_source_s3'] + "-" + stage;
+			}
+			def s3Exists = true;
+			try {
+				sh "aws s3api head-bucket --bucket ${targetBucket} --output json"
+			} catch (ex) {
+				echo "Bucket does not exist"
+				s3Exists = false
+			}								
+			if(s3Exists) {
+				def output = sh(
+					script: "aws s3 ls s3://${targetBucket} --recursive | wc -l",
+					returnStdout: true
+				).trim()				
```

Comment:
There are existing functions for checking bucket exist or not and for the bucket is empty or not. Kindly reuse the existing methods.

Reply:
updated